app/graphql/resolvers/customers/subscriptions_resolver.rb

```diff
@@ -12,9 +12,17 @@ class SubscriptionsResolver < Resolvers::BaseResolver
       type Types::Subscriptions::Object, null: false
 
       def resolve(status: nil)
+        statuses = status
         subscriptions = object.subscriptions
-        subscriptions = subscriptions.where(status: status) if status.present?
-        subscriptions.order(created_at: :desc)
+
+        return subscriptions.order(created_at: :desc) if statuses.blank?
```

Comment:
It's not really clear what we are trying to achieve here. Do you think we could refact it a bit or at least add some comments?

For example `order(created_at: :desc)` is shared between all returns

Reply:
My first approach was with shared ordering for instance, but in that case I had lot of if else and IMO it was even more messier. Added comments for now but if I think of some more simple solution, I will implement it for sure 👍